pyfar/dsp/dsp.py

```diff
@@ -1393,3 +1393,39 @@ def time_shift(signal, shift, unit='samples'):
             axis=-1)
 
     return shifted.reshape(signal.cshape)
+
+
+def convolve_overlap_add(signal1, signal2, mode='full'):
```

Comment:
I would prefer only 'convolve()' for brevity. We could still clearly mention in the docstring that we are wrapping scipy.oaconvolve. I assume, we are not going to introduce another function for convolution and do not need to be explicit in the function name.